### PR TITLE
docs: add AuthPage doc for mantine

### PR DESCRIPTION
**Pull request by yildirayunlu**
Please provide enough information so that others can review your pull request:

Add AuthPage doc for mantine.

### CODE REVIEW

File: .changeset/hot-avocados-begin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@pankod/refine-mantine": minor
+---
+
+Add formProps prop on AuthPages.
```
Comment (omeraplak) [inline .changeset/hot-avocados-begin.md:5]:
```suggestion
Updated `formProps` property on `<AuthPage />` component
```

File: examples/authentication/mantine/src/App.tsx [hunk 2]
```diff
@@ -134,6 +144,22 @@ const App: React.FC = () => {
...
+                                    const password = e.target.password.value;
+
+                                    alert(
+                                        JSON.stringify({
+                                            email,
+                                            password,
+                                        }),
+                                    );
+                                },
+                            }}
```
Comment (omeraplak) [inline examples/authentication/mantine/src/App.tsx:162]:
```suggestion
```

File: examples/authentication/mantine/src/App.tsx [hunk 3]
```diff
@@ -77,6 +79,14 @@ const App: React.FC = () => {
...
     };
 
+    const RememberMe = () => {
+        const { getInputProps } = FormContext.useFormContext();
+
+        return (
+            <Checkbox label="Remember Me" {...getInputProps("rememberMe")} />
+        );
+    };
+
```
Comment (omeraplak) [inline examples/authentication/mantine/src/App.tsx:89]:
```suggestion
```

File: packages/mantine/src/components/pages/auth/components/forgotPassword/index.tsx [hunk 4]
```diff
@@ -41,23 +39,27 @@ export const ForgotPasswordPage: React.FC<ResetPassworProps> = ({
...
     const { Link } = useRouterContext();
 
-    const { getInputProps, onSubmit } = useForm({
+    const form = useForm({
         initialValues: {
             email: "",
         },
         validate: {
-            email: (value) =>
+            email: (value: any) =>
```
Comment (omeraplak) [inline packages/mantine/src/components/pages/auth/components/forgotPassword/index.tsx:52]: could we use strict type?

File: packages/mantine/src/components/pages/auth/components/forgotPassword/index.tsx [hunk 5]
```diff
@@ -66,56 +68,67 @@ export const ForgotPasswordPage: React.FC<ResetPassworProps> = ({
...
-                    fullWidth
-                    size="md"
-                    type="submit"
-                    loading={isLoading}
-                >
-                    {translate(
-                        "pages.resetPassword.buttons.submit",
-                        "Send reset instructions",
+                                    "pages.login.forgotPassword.haveAccount",
+                                    "Have an account? ",
```
Comment (omeraplak) [inline packages/mantine/src/components/pages/auth/components/forgotPassword/index.tsx:103]:
```suggestion
                                    "Have an account?",
```

File: packages/mantine/src/components/pages/auth/components/login/index.tsx [hunk 6]
```diff
@@ -44,26 +38,30 @@ export const LoginPage: React.FC<LoginProps> = ({
...
-    const { getInputProps, onSubmit } = useForm({
+    const form = useForm({
         initialValues: {
             email: "",
             password: "",
             remember: false,
         },
         validate: {
-            email: (value) =>
+            email: (value: any) =>
```
Comment (omeraplak) [inline packages/mantine/src/components/pages/auth/components/login/index.tsx:53]:
```suggestion
            email: (value: any) =>
```
could we use strict type? @salihozdemir @yildirayunlu

**Review by omeraplak — CHANGES_REQUESTED** [review]

File: packages/mantine/src/components/pages/auth/components/register/index.tsx [hunk 7]
```diff
@@ -14,103 +14,155 @@ import {
...
+                        )}
+                        placeholder="●●●●●●●●"
+                        {...getInputProps("password")}
+                    />
+                    {loginLink ?? (
+                        <Group mt="md" position={loginLink ? "left" : "right"}>
+                            <Text size="xs">
+                                {translate(
+                                    "pages.register.buttons.haveAccount",
+                                    "Have an account? ",
```
Comment (omeraplak) [inline packages/mantine/src/components/pages/auth/components/register/index.tsx:140]:
```suggestion
                                    "Have an account?",
```

File: packages/mantine/src/components/pages/auth/components/login/index.tsx (hunk 6 above)
Comment (yildirayunlu) [reply packages/mantine/src/components/pages/auth/components/login/index.tsx:53]: `FormType` interface can be passed as `createFormContext` generic type. However, we cannot do this in this use. 

We can come up with a different solution to this later.

**Review by omeraplak — APPROVED** [review]

**Pull request merged**